invenio_damap/assets/semantic-ui/js/invenio_damap/index.js

```diff
@@ -8,18 +8,37 @@
 import ReactDOM from "react-dom";
 import React from "react";
 import { Grid } from "semantic-ui-react";
-import { DMPButton } from "./DMPButton";
 
-const element = document.getElementById("invenio-damap-render");
+import { DMPAddButton } from "./DMPAddButton";
+import { DMPAuthButton } from "./DMPAuthButton";
 
-const recordManagementAppDiv = document.getElementById("recordManagement");
-const record = JSON.parse(recordManagementAppDiv.dataset.record);
+const element = document.getElementById("dmps-container");
 
-// the render element won't be available if we're not the record owner
-element &&
+if (element) {
+  const availableDmps = element.getAttribute("data-dmps");
+  const damapUrl = element.getAttribute("data-damap-url");
+
+  const parsedAvailableDmps =
+    availableDmps && availableDmps !== "null" ? JSON.parse(availableDmps) : [];
```

Comment:
do we need an explicit null check? might `availableDmps` as a check be enough?
![image](https://github.com/user-attachments/assets/511ce3ec-4ef4-42c6-ab46-e12b3d37571a)

Reply:
@max-moser I kept your suggestion but note that prettier still refactors it without the parentheses.

Reply:
@rekt-hard You're right, we're not passing any null from the backend, this was kept in case I figured out how to return null for no identity present in DAMAP vs an empty list for no DMPs.